Suggest convention for /livez and /readyz (#70)
nodeshift/nodejs-reference-architecture: docs/operations/healthchecks.md

### Endpoints

> We recommend using `/readyz` and `/livez` as the endpoints for the readiness and

Reviewer: ```suggestion
We recommend that you use consistent naming for your endpoints across micro-services.  `/readyz` and `/livez` are common choices for the endpoints for the readiness and
```

Reviewer: Discussed in Ref Arch team meeting today, we are doing a bit more research and will comment soon.

liveness probes, respectively.

Any route name can work if it agrees with the probe configuration, but in the
absence of some requirement by the tooling, best to use names that have a
clear relationship to their purpose. `/healthz` is not clear as to whether it is
liveness or readiness.  Because
[the differences between them are important](https://developers.redhat.com/blog/2020/11/10/you-probably-need-liveness-and-readiness-probes),
it's best to clarify.

The "z" at the end of `/readyz` and `/livez` are a pattern called "z pages" that

> is [used by Kubernetes](https://kubernetes.io/docs/reference/using-api/health-checks/)

Reviewer: ```suggestion
is a pattern used by [internal Kubernetes services](https://kubernetes.io/docs/reference/using-api/health-checks/) and has also been adopted by some other projects like OpenCensus.
```

> itself.

Reviewer: ```suggestion
```
